### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by muhenge**
**Work done**

-  We added luxon package for date implementation
-  Added all navigation in the navbar
- Add new books HTML page
- Added the contact page
- Added the copyright footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,16 +4,48 @@
...
+        <div class="collapse navbar-collapse mr-auto col-lg-3 justify-content-around" id="navbarText">
+          <ul class="navbar-nav">
+            <li class="nav-item active">
+              <a class="nav-link" href="./index.html">List</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./add-new.html">Add New</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./contact.html">Contact</a>
```
Comment (sinansevgi) [inline index.html]: - [x] You should **not** have different pages for these sections. If you check the project requirement, you can see that you need to manipulate the content of the page with javascript in order the change sections. (When I click on the link only the content of the main area should change, not the page.)
You can simply add (classList.add) and remove (classList.remove) CSS classes to display or hide the content.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @muhenge and @Felipeg005 , 

Good job so far!
Great work with using luxon to get date :+1: 
There are some issues that you still need to work on to go further but you are almost there!

 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Felipeg005) [reply index.html]: Hi sinansevgi, ok perfect you are totally right, Now we fix the code to manipulate the content of the page with javascript in order the change sections :smile: .

Comment (rloterh) [reply index.html]: This issue is well resolved! ✔️

File: index.html [hunk 2]
```diff
@@ -4,16 +4,63 @@
...
+            <li class="nav-item">
+              <a id="list-nav" onclick="show('list-section')" class="nav-link">List</a>
+            </li>
+            <li class="nav-item">
+              <a id="add-nav" onclick="show('form-section')" class=" nav-link">Add New</a>
+            </li>
+            <li class="nav-item">
+              <a id="contact-nav" onclick="show('contact-section')" class=" nav-link">Contact</a>
+            </li>
+          </ul>
```
Comment (rloterh) [inline index.html:28]: - [ ] OPTIONAL
- The right menu elements in your navbar is missing the dividers between each menu element, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.
- To enhance the user experience of this design kindly ensure that an active menu link is highlighted(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted.

**Your design menu:**
image

**The wireframe menu design:**
image

File: index.html [hunk 3]
```diff
@@ -4,16 +4,63 @@
...
+            <li class="list-group-item h4 col">Do you have any questions or you just want to say hello?<br>
+            you can reach out to us!</li>
+            <li class="list-group-item h5 col">Our e-mail: awesomebooks@gmail.com</li>
+            <li class="list-group-item h5 col">Our phone number +1 (423) 7856</li>
+            <li class="list-group-item h5 col">Our adresse: MIAMI FL 320456 str road Drv-Okland</li>
+          </ul>
+        </section>
+    </main>
+    <footer class="footer d-flex justify-content-center text-center pt-4 pb-5 mt-2 vw-100">
+      © 2021 Copyright:
```
Comment (rloterh) [inline index.html:60]: - [ ] OPTIONAL
Again, as can be seen in the wireframe template, let's ensure that the `Copyright` is aligned to the left of the footer section

File: index.html [hunk 4]
```diff
@@ -4,16 +4,63 @@
...
+        <section id="contact-section" class="d-none">
+          <h1 class="text-center mb-3 m-4">Contact</h1>
+          <ul class="contact list-group p-5 row align-items-center justify-content-center">
+            <li class="list-group-item h4 col">Do you have any questions or you just want to say hello?<br>
+            you can reach out to us!</li>
+            <li class="list-group-item h5 col">Our e-mail: awesomebooks@gmail.com</li>
+            <li class="list-group-item h5 col">Our phone number +1 (423) 7856</li>
+            <li class="list-group-item h5 col">Our adresse: MIAMI FL 320456 str road Drv-Okland</li>
+          </ul>
+        </section>
```
Comment (rloterh) [inline index.html:57]: - [ ]  OPTIONAL
- The contents of your _contact page_ does match the provided wireframe design template. As we want our design to as much as possible closely follow the the provided [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md) kindly ensure to align `Our email`, ` Our contact` and `Our address` to the **left**, and **bulleted** as can be observed in the template design

**Your design:**
image

**The wireframe design:**
image

**Review by rloterh — APPROVED** [review]
Hi Herve & Juan, 

Great job for making the necessary changes recommended by the previous reviewer. However, there's always room for improvement.
Kindly consider my OPTIONAL comments

## Optional suggestions
_Please keep in mind that you, for best practices and following the project requirements and the design templates closely kindly consider these aspects as outlined in the below comments._

**_Kindly notice that these are required changes that I'm sorry they didn't show up in the earlier review._**

Also, kindly consider:
- [ ] OPTIONAL:
- Kindly note, the correct github flow should be: `merge commits into main from feature_branch`
- Kindly ensure to update your ReadMe with details of the current project so as to make it more professional.

<br>
Your project is complete! Congratulations! 🥂There is nothing else to say other than... it's time to merge it 👍.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**